tests/utils/test_utils.py

```diff
@@ -156,3 +157,92 @@ def test_rotate_cell(self):
 
         newdoc.validate(recursive=True)
         write_neuroml2_file(newdoc, "tests/utils/test_rotation.net.nml", validate=True)
+
+    def test_translate_cell_to_coords(self):
+            """Test translate_cell_to_coords"""
+            acell = neuroml.utils.component_factory(
+                "Cell", id="test_cell", validate=False
+            )  # type: neuroml.Cell
+
```

Comment:
add a spike threshold here, with the latest release of the schema (currently in the dev branches), spike thresh will become a compulsory element:

```
acell.set_spike_thresh("10mV")
```

Reply:
Should I also add it to the test_rotate_cell function?

Reply:
After adding that and running the test I get some warnings like the following, is it normal?
tests/utils/test_utils.py::TestUtils::test_translate_cell_to_coords                                                       
:..site-packages\neuroml\nml\generatedssupersuper.py:176: UserWarning: membrane_properties has already been assigned.  Use `force=True` to overwrite. Hint: you can make changes to the already added object as required without needing to re-add it because only references to the objects are added, not their values.

Reply:
Yes, that's fine. It's a message we give to the user to tell them that a part of the model already exists and we didn't overwrite it